.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -78,6 +78,15 @@ body:
           required: false
     validations:
       required: true
+  - type: "textarea"
+    attributes:
+      label: "下载器版本号 - Downloader Version"
+      description: |-
+        Enter the PBH version that display on WebUI footer or GUI window title.  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)